apps/console/src/features/server-configurations/utils/governance-connector-utils.ts

```diff
@@ -171,55 +174,52 @@ export class GovernanceConnectorUtils {
         });
     }
 
-    public static getPredefinedConnectorCategories() {
+    public static getPredefinedConnectorCategories(): Array<any> {
```

Comment:
Check whether we can use `GovernanceConnectorCategoryInterface[]` other than `any`